src/connectors/current-refinements/connectCurrentRefinements.ts

```diff
@@ -297,7 +348,7 @@ function normalizeRefinement(refinement: Refinement): ConnectorRefinement {
       )} ${refinement.name}`
     : refinement.name;
 
-  const normalizedRefinement: ConnectorRefinement = {
+  const normalizedRefinement: CurrentRefinementsConnectorParamsRefinement = {
```

Comment:
Previously we also exported a normalised type, I decided to remove it to make things simple.